src/mame/matsushita/kn5000.cpp

```diff
@@ -836,15 +833,8 @@ ROM_START(kn5000)
 	ROMX_LOAD("kn5000_subprogram_v140.rom", 0x000000, 0x030000, CRC(d9a537aa) SHA1(b7f471522ab3125e5eb42c7368d57a56084ce32a), ROM_BIOS(5)) // v5
 	ROMX_LOAD("kn5000_subprogram_v139.rom", 0x000000, 0x030000, NO_DUMP, ROM_BIOS(6)) // v4
 
-	ROM_REGION16_LE(0x20000, "mask", 0) // subcpu boot rom
-	ROM_LOAD("kn5000_mask_rom.ic30", 0x00000, 0x20000, NO_DUMP)
-	// hack to keep the CPU from touching SFRs arbitrarily while we do not have a proper ROM dump:
-	ROM_FILL(0x000000, 1, 0x68) // 68 fe = infinite loop
-	ROM_FILL(0x000001, 1, 0xfe)
-	ROM_FILL(0x01ff00, 1, 0x00) // RESET vector = 0x00fe0000
-	ROM_FILL(0x01ff01, 1, 0x00)
-	ROM_FILL(0x01ff02, 1, 0xfe)
-	ROM_FILL(0x01ff03, 1, 0x00)
+	ROM_REGION16_LE(0x20000, "mask", 0)
```

Comment:
Also the tradition is to name a rom region with what it's used for.  So here "subcpu" would make more sense than the horribly generic "mask".